packages/pentaho/package.json

```diff
@@ -44,13 +44,15 @@
     "@hitachivantara/uikit-react-icons": "^5.12.0",
     "@hitachivantara/uikit-react-utils": "^0.2.5",
     "@mui/base": "^5.0.0-beta.40",
-    "react-resize-detector": "^8.1.0"
+    "react-resize-detector": "^8.1.0",
+    "sanitize-html": "^2.13.0"
```

Comment:
this library is really bloated/overkill for what we need - can't we use something else?

Reply:
I'll look into it 👍